doc/ConnectionPooling.md

```diff
@@ -0,0 +1,387 @@
+## Using Connection Pools
+
+### Multiple Connection Pools
+
+Snowflake .NET Driver v4.0.0 provides multiple pools with couple of additional features in comparison to the previous implementation.
+Each pool is identified by the entire <b>connection string</b>. Order of connection string parameters is significant and the same connection parameters
```

Comment:
I would open the new paragraph with this sentence. The previous sentence was about differences with the old pool - now when we write about the order of parameters - the order was important also in the previous pool. Giving a new paragraph makes it clear that we are not talking about differences.